src/wallet/wallet_rpc_server.cpp

```diff
@@ -4306,21 +4306,31 @@ namespace tools
       return false;
     }
 
-    return fill_response(ptx_vector,
-                         req.get_tx_key,
-                         res.tx_key,
-                         res.amount,
-                         res.fee,
-                         res.multisig_txset,
-                         res.unsigned_txset,
-                         req.do_not_relay,
-                         false /*blink*/,
-                         res.tx_hash,
-                         req.get_tx_hex,
-                         res.tx_blob,
-                         req.get_tx_metadata,
-                         res.tx_metadata,
-                         er);
+    try
+    {
+      fill_response(ptx_vector,
```

Comment:
Should this just be returning the `fill_response` value?  Is it possible for `fill_response` to return false?  (And if not, just make it a void function).

Reply:
Should be returning, matches the other code.